components/dropdown/dropdown-content.js

```diff
@@ -1,23 +1,41 @@
 import { DropdownContentMixin } from './dropdown-content-mixin.js';
 import { dropdownContentStyles } from './dropdown-content-styles.js';
+import { DropdownPopoverMixin } from './dropdown-popover-mixin.js';
 import { LitElement } from 'lit';
 
-/**
- * A generic container for dropdown content.  It provides behavior such as sizing,  positioning, and managing focus gain/loss.
- * @slot - Anything inside of "d2l-dropdown-content" that isn't in the "header" or "footer" slots appears as regular content
- * @slot header - Sticky container at the top of the dropdown
- * @slot footer - Sticky container at the bottom of the dropdown
- * @fires d2l-dropdown-open - Dispatched when the dropdown is opened
- */
-class DropdownContent extends DropdownContentMixin(LitElement) {
-
-	static get styles() {
-		return dropdownContentStyles;
-	}
+const usePopoverMixin = window.D2L?.LP?.Web?.UI?.Flags.Flag('GAUD-7472-dropdown-popover', false);
```

Comment:
I think how you have it could result in `usePopoverMixin` being `undefined` if `D2L...` isn't there. Typically we do this:

```suggestion
const usePopoverMixin = window.D2L?.LP?.Web?.UI?.Flags.Flag('GAUD-7472-dropdown-popover', false) ?? true;
```

Also, since this is repeated a bunch of times could it maybe be exported from somewhere central and then imported everywhere else?

Reply:
That's a good point. Currently I think we will want to fall back to `false` since I'm not sure we want to turn this on before we've done the (LMS) clean-up for the fixed-positioing flag.

Reply:
And yep, I can move this somewhere shared.

Reply:
We could even start a `flags.js` helper somewhere central in core. i.e.

```javascript
import { usePopoverMixin } from '../../helpers/flags.js';
```

Reply:
Oh yeah they'll all be done correctly, unlike mine here. 😏

Reply:
I've defined the `getFlag` helper as discussed offline. I added basic tests to cover the cases where the API is either defined or not. Also, I did not document it in the readme, since I'm not sure we want anyone else using this.